nncf/common/pruning/symbolic_mask.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, mask_producers: List[int] = None):
         super().__init__(-1, mask_producers)
 
 
-class SymbolicMaskProcessor(NNCFBaseTensorProcessor):
+class SymbolicMaskProcessor(NNCFPruningBaseTensorProcessor):
```

Comment:
Why we need base class `NNCFPruningBaseTensorProcessor` which is base only for one class `SymbolicMaskProcessor`?